crates/engine/tree/src/chain.rs

```diff
@@ -189,7 +189,7 @@ pub trait ChainHandler: Send + Sync {
 }
 
 /// Events/Requests that the [`ChainHandler`] can emit to the [`ChainOrchestrator`].
-#[derive(Clone, Debug)]
+#[derive(Debug)]
```

Comment:
we can still have `Clone` here right?

Reply:
Yeah absolutely, that was from the previous change but I was curious what you would say about it.